BitwardenShared/UI/Vault/VaultItem/AddEditItem/AddEditItemView.swift

```diff
@@ -58,7 +65,17 @@ struct AddEditItemView: View {
     }
 
     private var content: some View {
-        ScrollView {
+        GuidedTourScrollView(
+            store: store.child(
+                state: { state in
+                    state.guidedTourViewState
+                },
+                mapAction: { action in
+                    .guidedTourViewAction(action)
+                },
```

Comment:
🎨 You can simplify this:
```suggestion
            store: store.child(
                state: \.guidedTourViewState,
                mapAction: AddEditItemAction.guidedTourViewAction,
```